docs/modding-and-development/development/contributing-code-to-northstar.md

```diff
@@ -30,3 +30,51 @@ To get the cropped GIFs of a screenrecording:
 3. Use a site like [https://ezgif.com/video-to-gif](https://ezgif.com/video-to-gif) for cropping and turning it into a GIF \
    (The linked site has not been vetted, use at own risk)
 4. Upload GIF to GitHub pull request description by just pasting it at the appropriate location.
+
+### Avoiding "PR hell"
+
+_"PR hell"_ is what contributors refer to when a pull request is stuck as pending without any indication for progress.
+
+There can be a variety reasons for this. Whether it's cause the pull request was overlooked, the changes are too big that anyone actually dares looking at it, or simply every reviewer is too busy at the moment, not getting a response on something you put a lot of work into is a terrible experience.
+
+While we are continuously working on improving the expierence and recruiting more reviewers, here's a few tips to avoid _"PR hell"_.
```

Comment:
```suggestion
While we are continuously working on improving the experience and recruiting more reviewers, here's a few tips to avoid _"PR hell"_.
```